subprojects/docs/src/docs/userguide/writing_build_scripts.adoc

```diff
@@ -42,29 +42,34 @@ Don't forget that your build script is simply Groovy code that drives the Gradle
 
 Let's try this out and try to access the `name` property of the `Project` object.
 
-++++
-<sample xmlns:xi="http://www.w3.org/2001/XInclude" id="projectApi" dir="userguide/tutorial/projectApi" title="Accessing property of the Project object">
-    <sourcefile file="build.gradle" snippet="project-name"/>
-    <output args="-q check"/>
-</sample>
-++++
+=== Example: Accessing property of the Project object
+
+[source.multi-language-sample,groovy]
+.build.gradle
+----
+include::{samplesPath}/userguide/tutorial/projectApi/build.gradle[tag=project-name]
+----
+
+.Output of **`gradle -q check`**
+----
+> gradle -q check
+include::{samplesPath}/userguide/tutorial/projectApi/projectApi.out[]
+----
 
 Both `println` statements print out the same property. The first uses auto-delegation to the `Project` object, for properties not defined in the build script. The other statement uses the `project` property available to any build script, which returns the associated `Project` object. Only if you define a property or a method which has the same name as a member of the `Project` object, would you need to use the `project` property.
 
 [[sec:standard_project_properties]]
-==== Standard project properties
+=== Standard project properties
 
 The `Project` object provides some standard properties, which are available in your build script. The following table lists a few of the commonly used ones.
 
 .Project Properties
-[cols="a,a,a", options="header"]
+[%header%autowidth,compact]
 |===
-| Name
-| Type
-| Default Value
+| Name | Type | Default Value
 
 | `project`
-| api:org.gradle.api.Project[]
+| link:{groovyDslPath}/org.gradle.api.Project.html[Project]
 | The `Project` instance
```

Comment:
The backtick-quoted `Project` doesn't have background color inside the table as it is outside the table. I'm not sure if this is a known issue.

Reply:
Is this "Won't fix"?

Reply:
Monospaced code elements don't have a background color anymore. Going for a dotted underline instead, so this is expected. Sorry for not answering.

Reply:
Ok, I see now that many links like this have backticks around them to denote code, and that _does_ have a gray background. I guess we should just add some backticks here.